app/src/main/java/com/bottlerocketstudios/mapsdemo/ui/map/YelpViewModel.kt

```diff
@@ -17,26 +20,74 @@ class YelpViewModel : BaseViewModel() {
 
     // UI
     val yelpBusinessState: MutableStateFlow<List<Business>> = MutableStateFlow(emptyList())
-    val dallasLatLng: LatLng = LatLng(LATITUDE, LONGITUDE)
+    val googleMapsMarkersLatLng = yelpBusinessState.map { businessList ->
+        businessList.map { business ->
+            YelpMarker(
+                latitude = business.coordinates.latitude,
+                longitude = business.coordinates.longitude,
+                businessName = business.businessName
+            )
+        }
+    }
+    val selectedMarker: MutableStateFlow<YelpMarker> = MutableStateFlow(YelpMarker(latitude = 0.0, longitude = 0.0, businessName = ""))
+    val dallasLatLng: LatLong = LatLong(DEFAULT_LATITUDE, DEFAULT_LONGITUDE)
+    private var searchJob: Job? = null
 
     private companion object {
-        const val LATITUDE = 32.7767
-        const val LONGITUDE = -96.7970
+        const val DEFAULT_LATITUDE = 32.7767
+        const val DEFAULT_LONGITUDE = -96.7970
+        const val SEARCH_DELAY_MILLS = 300L
```

Comment:
@Josh-Gebbeken nit: I've seen `_MS` or `_MILLIS` much more frequently than `_MILLS` and would recommend choosing one of those :)